cypress/e2e/0-api/configuration_api.cy.js

```diff
@@ -0,0 +1,377 @@
+describe('configuration API', () => {
+  const PORT_REGEX = /[0-9]*/;
+
+  beforeEach(() => {
+    cy.login();
+  });
+
+  context('GET /config/network', () => {
+    it('retrieve network settings', () => {
+      cy.reqNetwork().then(response => {
+        const body = response.body;
+        expect(body.type).to.be.a('string');
+        if (body.type !== 'DHCP') {
+          expect(body).to.have.property('address');
+          if (body.address !== null) { expect(body.address).to.be.a('string'); }
+          expect(body).to.have.property('subnetMask');
+          if (body.subnetMask !== null) { expect(body.subnetMask).to.be.a('string'); }
+          expect(body).to.have.property('gateway');
+          if (body.gateway !== null) { expect(body.gateway).to.be.a('string'); }
+          expect(body).to.have.property('dns');
+          if (body.dns !== null) { expect(body.dns).to.be.a('string'); }
+        }
+      });
+    });
+  });
+
+  context('GET /config/hostname', () => {
+    it('retrieve hostname', () => {
+      cy.reqHostname().then(response => {
+        const body = response.body;
```

Comment:
This is more idiomatic modern javascript.
```suggestion
        const { body } = response;
```

Or you can do
```suggestion
        const { hostname } = response.body;
```